mitmproxy/contrib/kaitaistruct/gif.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, _io, _parent=None, _root=None):
         self.hdr = self._root.Header(self._io, self, self._root)
         self.logical_screen_descriptor = self._root.LogicalScreenDescriptorStruct(self._io, self, self._root)
         if self.logical_screen_descriptor.has_color_table:
-            self._raw_global_color_table = self._io.read_bytes((self.logical_screen_descriptor.color_table_size * 3))
+            self._raw_global_color_table = self._io.read_bytes(self.logical_screen_descriptor.color_table_size * 3)
```

Comment:
We probably want to exclude `contrib`? At least kaitaistruct classes are autogenerated.

Reply:
True - I'll revert for the whole sub-dir.